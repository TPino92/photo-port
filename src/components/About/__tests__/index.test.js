import React from 'react';
// react testing lib
import { render, cleanup } from '@testing-library/react';
// jest testing lib
import '@testing-library/jest-dom/extend-expect';
// import components
import About from '..';

afterEach(cleanup);

describe('About component', () => {
    //render About test
    // first test
    it('renders', () => {
        render(<About />);
    });
    // second test
    it('matches snapshot Dom node structure', () => {
        const { asFragment } = render(<About />);

        expect(asFragment()).toMatchSnapshot();
    })
})