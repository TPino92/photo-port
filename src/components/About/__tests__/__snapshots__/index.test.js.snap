// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot Dom node structure 1`] = `
<DocumentFragment>
  <section
    class="mx-5"
  >
    <h1
      id="about"
    >
      Who am I?!
    </h1>
    <img
      alt="cover"
      class="mx-2"
      src="cover-image.jpg"
      style="width: 100%;"
    />
  </section>
</DocumentFragment>
`;
